os/include/net/lwip/lwipopts.h

```diff
@@ -527,6 +527,19 @@
 #define MEMP_SEPARATE_POOLS	CONFIG_NET_MEMP_SEPARATE_POOLS
 #endif
 
+#if defined(CONFIG_STDK_IOT_CORE)
```

Comment:
lwipopt.h should be related only with the LwIP's own features, not STDK.

Reply:
new Kconfig item NET_LWIP_SINGLE_PBUF is added for this, please check the modification and share your opinion, thanks

Reply:
Got it.